clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -997,7 +1000,9 @@ class KubernetesApiConverter {
       labels += ["detail": parsedName.detail]
     }
 
-    labels += [(KubernetesUtil.SERVER_GROUP_LABEL): name]
+    if (!isNewController) {
```

Comment:
This is a bit of a strange way to handle this, will the other controllers always be considered "new"? I would at least rename this variable, or find another way to tag these.